locales/fr.json

```diff
@@ -155,5 +154,10 @@
   "some_income.title": "Votre situation",
   "no_income.title": "Votre situation",
   "unchanged_income.title": "Votre situation",
+<<<<<<< HEAD
+  "gross_income.title": "Votre revenu",
+  "results.banner": "Vous n’avez pas répondu à quelques questions. <a href='/clear'>Recommencez</a> pour recevoir vos résultats complets."
+=======
   "gross_income.title": "Votre revenu"
+>>>>>>> master
```

Comment:
Got some merge conflict markings committed

Reply:
hahah yup, just spotted it toggling language 🤦‍♀️, thank you!!!